### DISCUSSION TITLE
The stats of the user not equal to the user quota?

### DISCUSSION THREAD

**Discussion by zhucan** [rook/rook#10566] [Q&A]
**Run: "radosgw-admin user stats --uid=zhucan", output:**
{
    "stats": {
        "size": 2422934202,
        "size_actual": 2457624576,
        "size_utilized": 0,
        "size_kb": 2366147,
        "size_kb_actual": 2400024,
        "size_kb_utilized": 0,
        "num_objects": 10437
    },
    "last_stats_sync": "2022-07-08T03:36:18.783384Z",
    "last_stats_update": "2022-07-08T03:36:18.778456Z"
}

**Run: "radosgw-admin user info --uid=zhucan", output:**
{
    "stats": {
        "size": 2422934202,
        "size_actual": 2457624576,
        "size_utilized": 0,
        "size_kb": 2366147,
        "size_kb_actual": 2400024,
        "size_kb_utilized": 0,
        "num_objects": 10437
    },
    "last_stats_sync": "2022-07-08T03:36:18.783384Z",
    "last_stats_update": "2022-07-08T03:36:18.778456Z"
}
[root@rook-node01 /]# radosgw-admin user info --uid=zhucan 
{
    "user_id": "zhucan",
    "display_name": "zhucan",
    "email": "",
    "suspended": 0,
    "max_buckets": 10,
    "subusers": [],
    "keys": [
        {
            "user": "zhucan",
            "access_key": "xxxxxxxxxxxxxxxx",
            "secret_key": "xxxxxxxxxxxxxxxxxxxxxxx"
        }
    ],
    "swift_keys": [],
    "caps": [
        {
            "type": "buckets",
            "perm": "*"
        },
        {
            "type": "metadata",
            "perm": "*"
        },
        {
            "type": "usage",
            "perm": "*"
        },
        {
            "type": "users",
            "perm": "*"
        },
        {
            "type": "zone",
            "perm": "*"
        }
    ],
    "op_mask": "read, write, delete",
    "default_placement": "",
    "default_storage_class": "",
    "placement_tags": [],
    "bucket_quota": {
        "enabled": false,
        "check_on_raw": false,
        "max_size": -1,
        "max_size_kb": 0,
        "max_objects": -1
    },
    "user_quota": {
        "enabled": true,
        "check_on_raw": false,
        "max_size": 10000000000,
        "max_size_kb": 9765625,
        "max_objects": 10000
    },
    "temp_url_keys": [],
    "type": "rgw",
    "mfa_ids": []
}

I had synced the stats manually.

**Comment by zhucan** [comment]
The default time of the "stats" is 5 min?

**Reply by thotz** [reply]
Are mentioning even though user quota is set, since stats not synced properly it failed to enforce quota??. And default time for sync is 5mins??

**Comment by parth-gr** [comment]
`user_quota` contains the maximum value a user can contain,

And at the above `stats` you get the current stats of users which is less then the max.

**Reply by zhucan** [reply]
more than not less than
